app/steps/medicaid/citizen_member.rb

```diff
@@ -1,14 +0,0 @@
-module Medicaid
```

Comment:
did you mean to delete this step?

Reply:
Yup!

Looks like this is left over from some previous refactor and is duplicate code.
https://github.com/codeforamerica/michigan-benefits/blob/master/app/steps/medicaid/citizen_member.rb
https://github.com/codeforamerica/michigan-benefits/blob/master/app/steps/medicaid/intro_citizen_member.rb